app/components/publications/publication-sidebar.hbs

```diff
@@ -119,9 +119,14 @@
         <div class="auk-form-group">
           <Auk::Label>{{t 'decision-date'}}</Auk::Label>
           {{#if this.publicationFlow.case.subcases}}
-            <span class="auk-u-muted">
-              {{moment-format this.treatment.startDate "DD-MM-YYYY"}}
-            </span>
+            <Auk::ButtonLink {{on "click" this.navigateToAgendaItem }}>
```

Comment:
Idealiter is dit toch een echte link (die je in een nieuwe tab kan openen). Zie bv https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/97498b4479f3a4c63d6522f5d30e9733776dded8/app/components/agenda/agendaitem/agendaitem-case/agendaitem-titles.hbs#L21 voor een voorbeeld van een link waarvan onderdelen async geladen moeten worden.